fields/types/date/DateFilter.js

```diff
@@ -102,7 +102,7 @@ var DateFilter = React.createClass({
     });
   },
   switchBetweenActiveInputFields(e, day, modifiers) {
-    if (modifiers.indexOf('disabled') > -1) return;
+    if (modifiers.disabled === true) return;
```

Comment:
Add some check for modifiers to avoid js error like "cannot read property disable of undefined"
ex. if (modifiers?.disable) return;

Would modifiers.disable type always boolean ? If not, should we return if it's a string "true" ?

Reply:
Updated to check whether modifier is defined.

Since the name functions defined in `modifiers` should always returns a
boolean value according to
[documentation](https://github.com/gpbl/react-day-picker/blob/v2.4.0/docs/APIProps.md#modifiers),
I'd like to removes the boolean type checking (e.g. modifiers.disable === true) here.

Reply:
If @Aylie-Chou have different opinions about type checking criteria (or conventions we could follow), we can have further discussion here.